Makefile

```diff
@@ -45,3 +45,18 @@ test:
 
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing --migrations
+
+test-data-migration:
+	set -e ; \
+	BRANCH=$$(git rev-parse --abbrev-ref HEAD); \
+	LATEST_TAG=$$(git describe --abbrev=0 --tags); \
+	\
+	echo "# Loading some data at $$LATEST_TAG"; \
+	git checkout $$LATEST_TAG; \
+	rm -fr storage; \
+	make migrate; \
+	python manage.py loaddata --database=default test-data/data.json; \
```

Comment:
To me, `dummydata` is mostly for developers who want to test the Ideascube UI/UX with some reasonable data into it. As a result, reindexing the content after `dummydata` makes sense, because without it searching in the app does not work.

However, `test-data-migration` will mostly be used in the CI, and as manual sanity checks (e.g when releasing), but only to test the migrations, not to load some data for a developer to then test the UI/UX of Ideascube. I mean, the data being loaded is complete gibberish, nothing makes sense in the UI after loading it. 😄 

Therefore, I didn't find it necessary to reindex here, as it wouldn't really be useful, but it would make the CI jobs longer.

Unless you meant for us to test reindexation in the CI as well, which is an interesting thought on its own. /me goes and ponder on this a bit...